README.md

```diff
@@ -21,3 +21,12 @@
 * Build python package
 
   `pipenv run setup.py sdist bdist_wheel`
+
+## Test
+
+* From Runtime directory
+* From Runtime directory
+
+  `pytest`
+  or
+  `python -m pytest`
```

Comment:
This should be `pipenv run pytest`, no need for the direct python invocation, anyone who is going to do the stuff in this README is going to have to understand python at least that much.